ts/src/bitget.ts

```diff
@@ -1599,38 +1690,19 @@ export default class bitget extends Exchange {
         //         "data": "888291686266343424"
         //     }
         //
-        const result = {
-            'id': this.safeString (response, 'data'),
-            'info': response,
-            'txid': undefined,
-            'timestamp': undefined,
-            'datetime': undefined,
-            'network': undefined,
-            'addressFrom': undefined,
-            'address': undefined,
-            'addressTo': undefined,
-            'amount': undefined,
-            'type': 'withdrawal',
-            'currency': undefined,
-            'status': undefined,
-            'updated': undefined,
-            'tagFrom': undefined,
-            'tag': undefined,
-            'tagTo': undefined,
-            'comment': undefined,
-            'fee': undefined,
-        };
+        const result = this.parseTransaction (response, currency);
         const withdrawOptions = this.safeValue (this.options, 'withdraw', {});
         const fillResponseFromRequest = this.safeValue (withdrawOptions, 'fillResponseFromRequest', true);
         if (fillResponseFromRequest) {
+            result['type'] = 'withdrawal';
```

Comment:
I think that such sporadical `fillResponseFromRequest` implementations are not good. we should have them either somewhat at base level globally for any method, or nowhere. it's not ok to "have only for some exchanges only in some methods".